source/pintab_adafruit_kb2040.py

```diff
@@ -0,0 +1,49 @@
+from board import (
+    A3,
```

Comment:
Huh, `A`-pins? Well, thankfully the pintab exists just for that.

Reply:
Yes, I took all of them from the diagram provided as pinout